test/e2e/network/service.go

```diff
@@ -1474,6 +1474,13 @@ var _ = SIGDescribe("Services", func() {
 		// This container is an nginx container listening on port 80
 		// See kubernetes/contrib/ingress/echoheaders/nginx.conf for content of response
 		jig.RunOrFail(namespace, nil)
+		var err error
+		// Make sure acceptPod is running. There are certain chances that pod might be teminated due to unexpected reasons.
+		acceptPod, err = cs.CoreV1().Pods(namespace).Get(acceptPod.Name, metav1.GetOptions{})
+		framework.ExpectNoError(err, "Unable to get pod %s", acceptPod.Name)
+		gomega.Expect(acceptPod.Status.Phase).To(gomega.Equal(v1.PodRunning))
```

Comment:
We can replace here with `framework.ExpectEqual(acceptPod.Status.Phase, v1.PodRunning)`.
After merging https://github.com/kubernetes/kubernetes/pull/80785 this line will be blocked.

Reply:
Thanks for pointing. 
Done.